### DISCUSSION TITLE
Restrict folder permissions to only view a certain folder

### DISCUSSION THREAD

**Discussion by josias-r** [directus/directus#7688] [Archive]
I want to create a user who can only see files inside one specific folder. The user should also not be able to see other folders in the list.

I tried the following filter for the folder read permissions:

```

{
    "id": {
        "_eq": "xxx-xxx-xx-xxx-xxxxxxx"
    }
}

```

Unfortunately the user still sees all other folders for some reason. I was at least able to hide all other files with similar filters for the file permissions.

Does anyone know how I could hide the folders as well?

**Comment by rijkvanzanten** [comment]
Your example seems to work for me:

Permissions for read on `directus_folders` the public role:

```json
{
    "id": {
        "_eq": "422059ef-5832-4dc0-9691-106dfe10ae64"
    }
}
```

Admin request:

![CleanShot 2021-08-30 at 11 35 57@2x](https://user-images.githubusercontent.com/9141017/131365185-f3b93f3f-eb81-453a-96c4-2a0a21a78d69.png)

Public request:

![CleanShot 2021-08-30 at 11 36 06@2x](https://user-images.githubusercontent.com/9141017/131365199-4c2b2a33-c342-4a4e-936e-bd926e204c85.png)

**Reply by josias-r** [reply]
Is it possible that the interface behaves differently than the API request? 🤔

**Reply by josias-r** [reply]
Actually, after further playing around, it seems that the folders were cached from the first user that was logged in with admin permissions. After I refreshed, the folders disappeared 🤦‍♂️

**Reply by rijkvanzanten** [reply]
Gotcha!
